doc/manual/src/command-ref/files/channels.md

```diff
@@ -1,26 +1,41 @@
 ## Channels
 
-A directory containing symlinks to Nix channels, managed by [`nix-channel`]:
-
-- `$XDG_STATE_HOME/nix/profiles/channels` for regular users
-- `$NIX_STATE_DIR/profiles/per-user/root/channels` for `root`
-
-[`nix-channel`] uses a [profile](@docroot@/command-ref/files/profiles.md) to store channels.
-This profile contains symlinks to the contents of those channels.
+Channels are a mechanism for obtaining Nix expressions over the web.
+
+The moving parts of channels are:
+- The official channels listed at <https://nixos.org/channels>
+- The user-specific list of [subscribed channels](#subscribed-channels)
+- The [downloaded channel contents](#channel-contents)
+- The [Nix expression search path](@docroot@/command-ref/conf-file.md#conf-nix-path)
+
+> **Note**
+>
+> The state of a subscribed channel is external to the Nix expressions relying on it.
+> This may limit reproducibility.
+>
+> Dependencies on other Nix expressions can be declared explicitly with:
+> - [`fetchurl`](@docroot@/language/builtins.md#builtins-fetchurl), [`fetchTarball`](@docroot@/language/builtins.md#builtins-fetchTarball), or [`fetchGit`](@docroot@/language/builtins.md#builtins-fetchGit) in Nix expressions
+> - the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) in command line invocations for explicitly determining the value of [lookup paths](@docroot@/language/constructs/lookup-path.md)
```

Comment:
```suggestion
> - the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) in command line invocations for explicitly setting the value of [lookup paths](@docroot@/language/constructs/lookup-path.md)
```

"Determine" can mean to observe and draw a conclusion.

If we need to be more vague

```suggestion
> - the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) in command line invocations for explicitly modifying the value of [lookup paths](@docroot@/language/constructs/lookup-path.md)
```

However, I now doubt that this should even be here, because command line invocations are transient - not _declaring explicitly_. A script might qualify, but the current text is about the command line (ie interactive).

Also you'd have to be explicit about what kind of `-I` value is actually qualifies as a precise pin for the purpose of eval reproducibility.

```suggestion
```

Reply:
Let's just leave the pointer here for now and address all of the details in a dedicated page.